Firmware/ultralcd.cpp

```diff
@@ -3456,8 +3441,7 @@ static void crash_mode_switch()
     {
         lcd_crash_detect_enable();
     }
-	if (printJobOngoing() || (lcd_commands_type == LcdCommands::Layer1Cal)) menu_goto(lcd_tune_menu, 9, true, true);
-	else menu_goto(lcd_settings_menu, 9, true, true);
+    menu_back();
```

Comment:
The call to `menu_back()` should be removed. There is no need for a menu change here :)